src/hotspot/share/gc/shenandoah/shenandoahGenerationalFullGC.cpp

```diff
@@ -117,10 +117,10 @@ void ShenandoahGenerationalFullGC::balance_generations_after_gc(ShenandoahHeap*
 
   if (old_capacity > old_usage) {
     size_t excess_old_regions = (old_capacity - old_usage) / ShenandoahHeapRegion::region_size_bytes();
-    gen_heap->generation_sizer()->transfer_to_young(excess_old_regions);
+    gen_heap->transfer_to_young(excess_old_regions);
```

Comment:
should we assert result is successful?  Or replace with force_transfer?  (just seems bad practice to ignore a status result)

Reply:
Yes, will try an assert here.